cmd/api-server/main.go

```diff
@@ -23,12 +27,32 @@ func init() {
 }
 
 func main() {
+
+  if os.Getenv("KUBECONFIG") == "" {
+		fmt.Println("KUBECONFIG not present. Exiting...")
+		os.Exit(1)
+	}
+
+	port := os.Getenv("APISERVER_PORT")
```

Comment:
indentation seems wrong!?

Reply:
no it's TAB - depends on display in editor/viewer :)